gazebo/Server.cc

```diff
@@ -69,6 +70,16 @@ namespace gazebo
 {
   struct ServerPrivate
   {
+    void InspectSDFElement(const sdf::ElementPtr _elem)
```

Comment:
Does this need the same checks as in #3017 ?
https://github.com/osrf/gazebo/blob/0219c47574def474db97355b3b0517aaabf37de5/gazebo/Server.cc#L87-L95

Reply:
gazebo9 only support SDF spec until 1.6 so we don't that need logic in this case.